src/ChooseList.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import "./ChooseList.css";
+
+function ChooseList(props) {
+    return <div>
+        <select
+            className="select"
+            value={props.id}
+            onChange={props.handleSelect}
+            aria-label="Hit space to enter lists, use cursor to select list, hit space to confirm"
+        >
+
+            {props.listOfLists.map(col =>
+                <option value={col.id}>{col.name}</option>)}
+        </select>
+
+        <span id={"buttons"}>
+        <button
+            className="editButton"
+            type={"button"}
+            id="editListName"
+            onClick={props.toggleEditListName}
+            aria-label="Hit space to edit name of list">
```

Comment:
No need to add "hit space to".  Voiceover user knows what they need to to do activate a button.